katdal/test/test_dataset.py

```diff
@@ -164,3 +165,9 @@ def test_uvw(self):
         assert_array_equal(self.dataset.u[:, 5], u)
         assert_array_equal(self.dataset.v[:, 5], v)
         assert_array_equal(self.dataset.w[:, 5], w)
+
+    def test_tag_selection(self):
+        self.dataset.select(target_tags='radec')
+        assert_equal(len(self.dataset.target_indices), 1)
```

Comment:
You should be able to compare the actual value of `target_indices`, not just the length.